client/src/App.tsx

```diff
@@ -5,14 +5,14 @@ import { io, Socket } from "socket.io-client";
 import axios from "axios";
 import { initializeApp } from "firebase/app";
 import { setPersistence, getAuth, inMemoryPersistence } from "firebase/auth";
-import { useLogin, LoadingScreen, AuthProvider, useAuth, apiUrl, Service } from "@hex-labs/core";
+import { useLogin, LoadingScreen, AuthProvider, useAuth, apiUrl, Service, Footer } from "@hex-labs/core";
 import { ChakraProvider } from '@chakra-ui/react'
 
 import "./App.css";
 import "./bulma-tooltip.min.css";
 import AdminHome from "./components/admin/AdminHome";
 // import ParticipantHome from "./components/participant/ParticipantHome";
```

Comment:
Small fix but please remove these imports that you've replaced with frontend-core components